docs/README.md

```diff
@@ -1,63 +1,101 @@
 [![Pillarbox logo](https://github.com/SRGSSR/pillarbox-apple/blob/main/docs/README-images/logo.jpg)](https://github.com/SRGSSR/pillarbox-android)
-[![GitHub releases](https://img.shields.io/github/v/release/SRGSSR/pillarbox-android)](https://github.com/SRGSSR/pillarbox-android/releases)
-[![android](https://img.shields.io/badge/android-21+-green)](https://github.com/SRGSSR/pillarbox-android)
-[![Build](https://github.com/SRGSSR/pillarbox-android/actions/workflows/build.yml/badge.svg?branch=main&event=push)](https://github.com/SRGSSR/pillarbox-android/actions/workflows/build.yml)
-[![GitHub license](https://img.shields.io/github/license/SRGSSR/pillarbox-android)](https://github.com/SRGSSR/pillarbox-android/blob/main/LICENSE)
 
-## About
+# Overview
 
-Pillarbox is the modern SRG SSR player ecosystem. It is build on top of [Media3 Exoplayer](https://developer.android.com/media/media3/exoplayer). So if you know how to work with
-_Exoplayer_, then you know how to work with _Pillarbox_.
+[![Last release](https://img.shields.io/github/v/release/SRGSSR/pillarbox-android?label=Release)](https://github.com/SRGSSR/pillarbox-android/releases)
+[![Android min SDK](https://img.shields.io/badge/Android-21%2B-34A853)](https://github.com/SRGSSR/pillarbox-android)
+[![Build status](https://img.shields.io/github/actions/workflow/status/SRGSSR/pillarbox-android/build.yml?label=Build)](https://github.com/SRGSSR/pillarbox-android/actions/workflows/build.yml)
+[![License](https://img.shields.io/github/license/SRGSSR/pillarbox-android?label=License)](https://github.com/SRGSSR/pillarbox-android/blob/main/LICENSE)
 
-## Compatibility
+Pillarbox is the modern SRG SSR multimedia player ecosystem, built on top of [AndroiX Media3](https://developer.android.com/media/media3). Pillarbox has been designed with robustness, flexibility, and  efficiency in mind, with full customization of:
+- Metadata and asset URL retrieval.
+- Asset resource loading, including support for Widevine and PlayReady.
+- Analytics and QoS integration.
+- User interface layout, in either [Compose](https://developer.android.com/develop/ui/compose/layouts) or [XML `View`s](https://developer.android.com/develop/ui/views/layout/declaring-layout).
 
-The library is suitable for applications running on android SDK 21 and above. The project is meant to be compiled with the latest Android version.
+Its robust player provides all essential playback features you might expect:
+- Audio and video (including 360° videos) playback.
+- Support for on-demand and live streams, with and without DVR.
+- Integration with the system playback user experience.
+- Playlist management, including bidirectional navigation.
+- Support for alternative audio tracks, audio description, subtitles, ...
+- Multiple instances support.
+- Picture-in-picture support.
```

Comment:
Supported but PIllarbox provide nothing on this specific topic.

Reply:
That's also true for others points in this list (alternative tracks, on-demand/live support, ...)

Reply:
Indeed, we can keep it then.